clients/admin-ui/src/types/api/models/CustomAssetType.ts

```diff
@@ -0,0 +1,10 @@
+/* istanbul ignore file */
+/* tslint:disable */
+/* eslint-disable */
+
+/**
+ * An enumeration.
+ */
+export enum CustomAssetType {
+  CUSTOM_FIDES_CSS = "custom-fides.css",
```

Comment:
I wasn't sure of our policy around `npm run openapi:generate` so I ran it to make sure this would autogenerate correctly and I removed the other autogenerated types.